src/runtime/vdom/test/update-element.spec.ts

```diff
@@ -71,7 +71,7 @@ describe('updateElement', () => {
 
   it('should add new class when no oldVNode', () => {
     const elm = document.createElement('my-tag') as HTMLElement;
-    const oldVNode: d.VNode = null;
+    const oldVNode: d.VNode | null = null;
```

Comment:
Yeah, I think we'd have to do the latter (type them as `null`), since they'd implicitly have `any` type (causing the compiler to yell at us) - done in https://github.com/ionic-team/stencil/pull/4905/commits/1cc54b7ca1676e5a29d3ceb0f44bd2ca6b0c1288